src/main/kotlin/org/taktik/icure/scheduledtask/PeriodicStorageTasksRequest.kt

```diff
@@ -0,0 +1,20 @@
+package org.taktik.icure.scheduledtask
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import org.taktik.icure.asynclogic.objectstorage.IcureObjectStorage
+
+private const val ONE_HOUR = 60 * 60 * 1000L
+
+@ExperimentalCoroutinesApi
+@Component
+class PeriodicStorageTasksRequest(
+	private val allObjectStorageLogic: List<IcureObjectStorage<*>>
+) {
+	@Scheduled(initialDelay = ONE_HOUR, fixedDelay = ONE_HOUR)
```

Comment:
Let's make it configurable : Add the ONE_HOUR as a property in ObjectStorageProperties